pkg/apis/crd/v1beta1/types.go

```diff
@@ -243,3 +243,80 @@ type IPPoolUsage struct {
 	// Number of allocated IPs.
 	Used int `json:"used"`
 }
+
+// AppliedTo selects the entities to which a policy is applied.
+type AppliedTo struct {
+	// Select Pods matched by this selector. If set with NamespaceSelector,
+	// Pods are matched from Namespaces matched by the NamespaceSelector;
+	// otherwise, Pods are matched from all Namespaces.
+	// +optional
+	PodSelector *metav1.LabelSelector `json:"podSelector,omitempty"`
+	// Select all Pods from Namespaces matched by this selector. If set with
+	// PodSelector, Pods are matched from Namespaces matched by the
+	// NamespaceSelector.
+	// +optional
+	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
+	// Groups is the set of ClusterGroup names.
+	// +optional
+	Groups []string `json:"groups,omitempty"`
+}
```

Comment:
This struct is shared with NetworkPolicy which contains more fields. Perhaps https://github.com/antrea-io/antrea/pull/5186 should be merged first as it contains the complete version of the struct.

Reply:
I will rebase after this PR is merged.